indico/modules/events/editing/controllers/backend.py

```diff
@@ -44,12 +49,44 @@ def _process(self):
 
 
 class RHEditingTags(RHEditingBase):
-    """Return all editing tags defined in the event."""
+    """Return all editing tags defined in the event or create a new tag."""
 
     def _process(self):
         return EditingTagSchema(many=True).jsonify(self.event.editing_tags)
 
 
+class RHCreateTag(RHEditingManagementBase):
+    @use_kwargs({
+        'code': fields.String(),
+        'title': fields.String(),
+        'color': fields.String(validate=validate.OneOf(get_sui_colors())),
+    })
+    def _process(self, code, title, color):
+        create_new_tag(self.event, code, title, color)
+        return EditingTagSchema(many=True).jsonify(self.event.editing_tags)
+
+
+class RHEditTag(RHEditingBase):
+    """Delete/Update tag."""
+
+    def _process_args(self):
+        RHEditingBase._process_args(self)
+        self.tag = EditingTag.query.with_parent(self.event).filter_by(id=request.view_args['tag_id']).first_or_404()
+
+    @use_args({
+        'code': fields.String(),
+        'title': fields.String(),
+        'color': fields.String(validate=validate.OneOf(get_sui_colors())),
+    })
+    def _process_PATCH(self, args):
+        update_tag(self.tag, args)
+        return EditingTagSchema(many=True).jsonify(self.event.editing_tags)
```

Comment:
Yeah, I agree. I just did it so that the frontend doesn't have to perform operations on the internal state. However I think that in the end it doesn't hurt and I will just return the updated object from this RH